lib/node_modules/@stdlib/stats/base/dists/planck/mean/test/fixtures/python/runner.py

```diff
@@ -0,0 +1,71 @@
+# !/usr/bin/env python
+#
+# @license Apache-2.0
+#
```

Comment:
Not clear why your runner diverges from our snippet: https://github.com/stdlib-js/stdlib/blob/develop/tools/snippets/test/fixtures/python/runner.py.